data/hai/unfettered side missions.txt

```diff
@@ -160,267 +96,413 @@ mission "Hai Legacy 1"
-		payment 1000000
-		log "Delivered a pressing food cargo to the Unfettered using a newly acquired but rather old Ladybug. They seem to not be firing on me, for now at least."
+		dialog `You receive a message from the associates you bought the ship from: "Thank you for helping our brethren, <last>. We will not forget this. Please accept these <payment>."`
+		payment 1500000
+		log "Delivered a pressing food shipment to the Unfettered Hai using an Anomalocaris. They're no longer shooting at me... for now, at least."
 
 
 
-# no category, meaning you can't remove it from the Ladybug
+# No category, meaning you can't remove it from the Anomalocaris
 outfit "Unfettered Landing Permit"
-	"maintenance costs" 681
+	"maintenance costs" 650
 	description "This item is needed in order to land on Unfettered planets, and its loss will cancel the arrangement."
 
 mission "Hai Legacy 3"
-	name "Ancient claim"
-	description "Beat the Lightning Bugs in a merciless fight. You only have one day, and being disabled will result in losing the fight."
 	landing
+	name "Ancient claim"
+	description "Beat the Lightning Bugs in combat. You only have one day, and becoming disabled will result in losing the fight."
 	source "Firelode"
-	deadline
+	deadline 2
 	to offer
-		has "flagship model: Ancient Ladybug"
+		has "flagship model: Anomalocaris"
 		has "Hai Legacy 2: done"
 	on offer
 		conversation
-			`As you land and exit your ship, you quickly notice you have attracted a lot of attention, probably due to its old design. One Unfettered that looks like an engineer comes to speak to you, warmer than usual. "Welcome. I do not know how you have obtained this ship, although I imagine some of us will see it as a sign. It is a relic of our past, a past we are quite proud of.`
-			`	"As such, most will say that you have no right to use that ship. Luckily for you, I do not believe in any of that. What matters to me is that this ship is very durable, given the time it obviously spent in the water. It also has ancient technology that would be very valuable to us."`
-			`	They raise their voice for everyone to hear, "I challenge you! The winner will earn the right to own this relic."`
+			`As you land, you realize your ship is the center of the spaceport crowd's attention, probably due to the ancient design. One Unfettered that looks like an engineer comes to speak to you, warmer than usual based on your experiences with the Unfettered. "Welcome. Although I do not know how you have obtained this ship, I imagine some of us will see it as a sign. It is a relic of our past, a past we are quite proud of.`
+			`	"As such, most will say that you need to prove to us you have a right to use that ship. Lucky for you, I don't believe in any of that. What matters to me is that this ship is very durable, and it's filled with valuable ancient technology, as well as Korath technology we have a right to salvage for."`
```

Comment:
```suggestion
			`	"As such, most will say that you need to prove to us you have a right to use that ship. Lucky for you, I don't believe in any of that. What matters to me is that this ship is very durable, and it's filled with valuable ancient technology, as well as Korath technology we have a right to salvage for."`
```
Is the last `for` needed? I'm not sure.

Reply:
I think it is? how else would you say it?

Reply:
I think grammatically they are both correct, but the meaning is subtly different. ~~It might just be my brain being weird, though.~~

`We have a right to salvage Korath technology` -> `Korath technology we have a right to salvage`

`We have a right to salvage for Korath technology` -> `Korath technology we have a right to salvage for`

I think the second ones mean more like "specifically look for korath tech to salvage", whereas the first one is more like "if it's korath tech we can salvage it"

I'm probably really overcomplicating it tbh.

Reply:
~~my brain thinks its fine~~